docs/criteria-for-promoted-release.md

```diff
@@ -0,0 +1,35 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Criteria for a Promoted Release
+
+Prior to bringing a project to the TSC for a vote on granting a Promoted
+Release, a project must fulfill the following criteria:
+
+1.  Project has reached active status (the processes and community are
```

Comment:
I think active should be capitalized: Active and be linked to its definition in the project lifecycle doc.